src/Access/LDAPClient.cpp

```diff
@@ -96,6 +96,39 @@ namespace
         return dest;
     }
 
+    auto escapeForFilter(const String & src)
+    {
+        String dest;
+        dest.reserve(src.size() * 2);
```

Comment:
It seems we replace some characters with three characters in the code below, so I think we should multiply `src.size()` by `3` here, not by `2`.

Reply:
Yes, you a right ), fixed